client/src/components/Map/index.tsx

```diff
@@ -1,8 +1,7 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import { graphql, useStaticQuery } from 'gatsby';
 import i18next from 'i18next';
-import { generateIconComponent } from '../../assets/icons';
+import { generateIconComponent, SuperBlock } from '../../assets/icons';
```

Comment:
Reuse type for icon

Reply:
SuperBlock is not a type. Please see comment about using global types. Otherwise, `superBlock` is just a string.

Reply:
I am bit confused here as generateIconComponent defines SuperBlock as a type and typescript complains when a string is assigned instead

Reply:
My bad, I missed the type def in the icon file. This should be fine. Thank you, for correcting me.

A future TODO should be to move that type def out of the icons/ folder

Reply:
Should this also move to redux/prop-types file from icons folder ? I can take that change in a separate PR

Reply:
> Should this also move to redux/prop-types file from icons folder ?

Not quite. There is a larger issue here, and I will open a detailed run-down of what should happen for this. Thank you.